t/http-router.t

```diff
@@ -33,6 +33,8 @@ throws-like { redirect 'foo' }, X::Cro::HTTP::Router::OnlyInHandler, what => 're
     'Can only use redirect term inside of a handler';
 throws-like { conflict }, X::Cro::HTTP::Router::OnlyInHandler, what => 'conflict',
     'Can only use conflict term inside of a handler';
+throws-like { i'm-a-teapot }, X::Cro::HTTP::Router::OnlyInHandler, what => "i'm-a-teapot",
+    'Can only use conflict term inside of a handler';
```

Comment:
A copy-pasto detected. :)

Reply:
Oh, thanks for pointing that out! Fixed the commit.